docs/api.md

```diff
@@ -615,19 +621,23 @@ Once the developer is done with merging records, conflicts are marked as
 resolved using the [`#resolve()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-resolve) method of the collection:
 
 ```js
-function sync() {
-  return articles.sync()
-    .then(res => {
-      if (res.ok)
-        return res;
-
-      // If conflicts, take remote version and sync again.
-      return Promise.all(res.conflicts.map(conflict => {
-        return articles.resolve(conflict, conflict.remote);
-      }))
-      .then(_ => sync());
-    })
-  .catch(console.error.bind(console));
+async function sync() {
+  try {
+    const res = articles.sync();
+
+    if (res.ok) {
+      return res;
+    }
+
+    // If conflicts, take remote version and sync again.
+    await Promise.all(res.conflicts.map(conflict => {
```

Comment:
shouldn't we add `async conflict => {` ?

Reply:
No need for async since there are no awaits inside this function, just regular promises that are returned and awaited with `await Promise.all`.